### PR TITLE
Feature/text key infrastructure

### PR DESCRIPTION
**Pull request by hedvigoscar**
`restring` will require some integration in the legacy app repo as well, we can hold off on it for now, however.

### CODE REVIEW

File: owldroid/src/main/java/com/hedvig/android/owldroid/service/TextKeys.kt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+        apolloClient
+                .query(textKeysQuery)
+                .enqueue(object : ApolloCall.Callback<Optional<TextKeysQuery.Data>>() {
+                    override fun onStatusEvent(event: ApolloCall.StatusEvent) {
+                        Timber.d("StatusEvent: %s", event.toString())
+                    }
+
+                    override fun onFailure(e: ApolloException) {
+                        Timber.d("Failed to load text keys :(")
```
Comment (palmfjord) [inline owldroid/src/main/java/com/hedvig/android/owldroid/service/TextKeys.kt]: should this really be a `.d`?

Comment (palmfjord) [reply owldroid/src/main/java/com/hedvig/android/owldroid/service/TextKeys.kt]: also maybe we should log out the exception?

File: app/src/main/java/com/hedvig/android/app/MarketingActivity.kt [hunk 2]
```diff
@@ -15,8 +17,8 @@ class MarketingActivity : DaggerAppCompatActivity() {
                 super.onAttachedToWindow()
                 val transaction = supportFragmentManager.beginTransaction()
                 transaction.add(
-                    R.id.marketing_screen_content,
-                    MarketingFragment()
+                        R.id.marketing_screen_content,
```
[the remark this answers is not in the document]
Comment (hedvigoscar) [reply app/src/main/java/com/hedvig/android/app/MarketingActivity.kt]: Indents are auto-generated by https://github.com/shyiko/ktlint

File: owldroid/src/main/java/com/hedvig/android/owldroid/service/TextKeys.kt (hunk 1 above)
Comment (hedvigoscar) [reply owldroid/src/main/java/com/hedvig/android/owldroid/service/TextKeys.kt]: Probably sound.

It should be noted that Timber.* does nothing on release builds currently. We may want to stash all timber messages into a buffer and attach it to Crashlytics if a crash occurs.

File: app/src/main/java/com/hedvig/android/app/MarketingActivity.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (palmfjord) [reply app/src/main/java/com/hedvig/android/app/MarketingActivity.kt]: this makes me sad) ):

[the remark this answers is not in the document]
Comment (hedvigoscar) [reply app/src/main/java/com/hedvig/android/app/MarketingActivity.kt]: 🤷‍♂️ it's like with `gofmt` and friends, it's really nobodies 100% preference 🙂

**Comment by hedvigoscar** [comment]
@palmfjord Can I merge this? 🙂

**Comment by hedvigoscar** [comment]
@palmfjord Ping 🙂

**Comment by palmfjord** [comment]
@hedvigoscar ALTHOUGH we shouldn't merge when CI is failing right?

**Comment by hedvigoscar** [comment]
@palmfjord I've fixed the issues blocking CI in another branch, would it be cool if I merge it anyways? 🙈

**Pull request merged**